README.md

```diff
@@ -18,11 +18,11 @@ TYPO3 V8: Extension Manager -> in2help -> Settings
 TYPO3 V9 or newer: Settings -> Extension Configuration -> in2help
 
 Options:
-* PID of Pattern Page: basic.designPatternPid (int+) -> the PID of the page which 
+* PID of Pattern Page: basic.designPatternPid (int+) -> the PID of the page which
 contains the Design Pattern. This a page in the page tree, which is the entry
 point to the editors documentation.
 * FE-Domain: basic.domain (text): Add a domain for the frontend rendering
-(optional if FE domain is different to BE domain - e.g. https://www.domain.org/)
+(optional if FE domain is different to BE domain - e.g. https://www.domain.org)
```

Comment:
When I look into the configuration, it seems that a trailing slash is needed here. See https://github.com/in2code-de/in2help/blob/master/ext_conf_template.txt